syntax = "proto2";

package foundation;

message ExtractedValue {
  required string id = 1;
  required string field_name = 2;
  required string type = 3;
  optional bytes serialized_value = 4;
  repeated string entity_ids = 5;
}

message ExtractedValueCollection {
  required string id = 1;
  repeated string extracted_value_ids = 2;
}